problems/4.arrays.js

```diff
@@ -12,15 +16,19 @@ function firstAndLastAverage() {
 * @param {number[]} arr - The input array
 * @returns {number[]} - The duplicated array
 */
-function duplicateArray() {
-
+const duplicateArray = (a, b) => {
+  let a = [1, 2, 3, 4, 5];
+  let b = [1, 2, 3, 4, 5];
+return (a.concat(b));
 }
 
 /** Removes the first 3 elements from an array. If the array has fewer than 3 elements, return an empty array.
 * @param {any[]} arr - The input array
 * @returns {any[]} - The array with the first 3 elements removed
 */
-function removeFirstThreeElements() {
+const removeFirstThreeElements = (arr = [ sugar, coffee, milk, tea]) => {
```

Comment:
No need for default argument, but inside of it sugar, coffee need to be strings 'sugar', 'coffee'